mutating-webhook/deployment/crds/moodle1.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: moodlecontroller.kubeplus/v1
+kind: Moodle
+metadata:
+  name: moodle1
+  # namespace: namespace1
+  annotations:
+    backup-volume: /opt/moodledata
+spec:
+  plugins: ["profilecohort"]
+  # <kind>.<instanceName>.<outputVariable>
+  mySQLServiceName: Fn::ImportValue(MysqlCluster.cluster1.service)
```

Comment:
Lets change this to include namespace.

Fn::ImportValue([Namespace].Kind.instanceName.outputVariable)

That is namespace can be optional.

Reply:
Ok